pkg/telemetry/cassandra_agent/cassandra_agent_config.go

```diff
@@ -22,10 +25,95 @@ import (
 var (
 	agentConfigLocation = "/opt/management-api/configs/metric-collector.yaml"
 	defaultAgentConfig  = telemetryapi.CassandraAgentSpec{
-		Endpoint: telemetryapi.Endpoint{
+		Endpoint: &telemetryapi.Endpoint{
 			Port:    "9000",
 			Address: "127.0.0.1",
 		},
+		Relabels: []promapi.RelabelConfig{
+			{
+				SourceLabels: []string{"__origname__"},
```

Comment:
Use ``__table__`` here instead and check for ``.+``. That way even with changes to original feed it would stay relevant.

Reply:
It appears that I actually needed to use `table`, not `__table__`, but once I made that change it appears that this works.